src/observable/Collection.js

```diff
@@ -97,6 +97,9 @@ define([
             this.onChanged.dispose();
             this._onItemChangedHandlerBound = null;
             this._collection = null;
+        },
+        toArray: function() {
+            return this._collection;
```

Comment:
Added this so that dart can extract just the list out of our special Collection object.